tests/test_vxlan_vtep_vni.rb

```diff
@@ -49,6 +49,8 @@ def test_create_with_existing
   end
 
   def test_vnis
+    skip('Platform does not support vnis') if VxlanVtepVni.vnis['nve1'].nil?
```

Comment:
The other testcases automatically skip because they attempt to `config_set` a property that is excluded for the platform.

The testcase assumed the config line in the setup method `config('interface nve1')` worked properly, which caused the following error for unsupported platforms:

```
  5) Failure:
TestVxlanVtepVni#test_vnis [tests/test_vxlan_vtep_vni.rb:53]:
Expected nil (NilClass) to respond to #empty?.
```